test/test_trixi.jl

```diff
@@ -17,8 +17,22 @@ as absolute/relative tolerance.
 macro test_trixi_include(elixir, args...)
     local l2 = get_kwarg(args, :l2, nothing)
     local linf = get_kwarg(args, :linf, nothing)
-    local atol = get_kwarg(args, :atol, 500 * eps())
-    local rtol = get_kwarg(args, :rtol, sqrt(eps()))
+    local RealT = get_kwarg(args, :RealT, :Float64)
+    if RealT === :Float64
+        atol_default = 500 * eps(Float64)
+        rtol_default = sqrt(eps(Float64))
+    elseif RealT === :Float32
+        atol_default = 500 * eps(Float32)
+        rtol_default = sqrt(eps(Float32))
+    elseif RealT === :Float128
+        atol_default = 500 * eps(Float128)
+        rtol_default = sqrt(eps(Float128))
+    elseif RealT === :Double64
+        atol_default = 500 * eps(Double64)
+        rtol_default = sqrt(eps(Double64))
+    end
```

Comment:
```suggestion
    local RealT_symbol = get_kwarg(args, :RealT, :Float64)
    RealT = getfield(@__MODULE__, RealT_symbol)
    atol_default = 500 * eps(RealT)
    rtol_default = sqrt(eps(RealT))
```
I think you can simplify this and make it more general. Didn't test it though.

Reply:
Good idea, I will give it a try!

Reply:
I have changed this in https://github.com/trixi-framework/TrixiShallowWater.jl/pull/96/commits/566f8c30918fd681e36a6416665eaebfd63d228e